cluster-provision/images/vm-image-builder/publish-containerdisk.sh

```diff
@@ -6,9 +6,20 @@ if [ "$#" -ne 2 ]; then
     echo "Run `publish-continerdisk.sh example quay.io/kubevirtci/example:mytag` to push the local `example:devel` image to `quay.io/kubevirtci/example:mytag`."
 fi
 
+get_image_tag() {
+    local current_commit today
+    current_commit="$(git rev-parse HEAD)"
+    today="$(date +%Y%m%d)"
+    echo "v${today}-${current_commit:0:7}"
+}
+
 export IMAGE_NAME=$1
 export TAG=devel
 export FULL_IMAGE_NAME=$2
 
+if [[ ! "$FULL_IMAGE_NAME" =~ ":" ]]; then
+  FULL_IMAGE_NAME=$FULL_IMAGE_NAME":"$(get_image_tag)
+fi
+
```

Comment:
I think its better to keep the tag of the final image as part of part of `FULL_IMAGE_NAME`
And generate the commit tag outside this script, on CI use case as part of the prow-job command
So the usage would be: 
```
# vm-image-builder prow-job
...
commit="$(git rev-parse HEAD)"
date="$(date +%Y%m%d)"
tag="v${today}-${current_commit:0:7}"
...
./cluster-provision/images/vm-image-builder/publish-containerdisk.sh "fedora" "quay.io/kubevirtci/fedora:${tag}
```